### PR TITLE
Data analytics

### PR DESCRIPTION
**Pull request by shrubs-ok**
New readme at the dataanalytics level
new Readme in the PowerBI folder
Place holders for the first pbix

### CODE REVIEW

File: MiCCSqlViewSampleQueries/AgentEventStats/README.md
Comment (shaapala) [inline MiCCSqlViewSampleQueries/AgentEventStats/README.md:1]: This was accidently removed when it needs to stay

File: MiCCSqlViewSampleQueries/AgentOutboundCallDetail/README.md
Comment (shaapala) [inline MiCCSqlViewSampleQueries/AgentOutboundCallDetail/README.md:1]: This was accidently removed when it needs to stay

**Review by shaapala — CHANGES_REQUESTED** [review]
Two readme files were deleted by accident. Need to bring them back

File: MiCCSqlViewSampleQueries/AgentEventStats/README.md
Comment (shrubs-ok) [reply MiCCSqlViewSampleQueries/AgentEventStats/README.md:1]: added file and committed

File: MiCCSqlViewSampleQueries/AgentOutboundCallDetail/README.md
Comment (shrubs-ok) [reply MiCCSqlViewSampleQueries/AgentOutboundCallDetail/README.md:1]: added file and committed

**Pull request closed without merging**